### PR TITLE
[Doc] syntax and format issues in README.md and how-to-contribute-to-apache-carbondata.md

### PR DESCRIPTION
**Pull request by Sunt-ing**
### Why is this PR needed?
 To improve the quality of README.md and how-to-contribute-to-apache-carbondata.md.
 
 ### What changes were proposed in this PR?
Syntax and format changes.
    
 ### Does this PR introduce any user interface change?
 - No 

 ### Is any new testcase added?
 - No

### CODE REVIEW

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/3625/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5370/

**Comment by Sunt-ing** [comment]
@Indhumathi27 Could you please review this PR? I believe this PR is very simple and thus will not take you much time to review. Thanks~

File: docs/how-to-contribute-to-apache-carbondata.md [hunk 1]
```diff
@@ -189,4 +189,4 @@ From another local branch, run:
 $ git fetch --all
 $ git branch -d <my-branch>
 $ git push <GitHub_user> --delete <my-branch>
-```
\ No newline at end of file
+```
```
Comment (Indhumathi27) [inline docs/how-to-contribute-to-apache-carbondata.md:192]: please revert this change

File: docs/how-to-contribute-to-apache-carbondata.md [hunk 2]
```diff
@@ -20,7 +20,7 @@
 The Apache CarbonData community welcomes all kinds of contributions from anyone with a passion for
 faster data format! Apache CarbonData is a new file format for faster interactive query using
 advanced columnar storage, index, compression and encoding techniques to improve computing
-efficiency,in turn it will help speedup queries an order of magnitude faster over PetaBytes of data.
+efficiency, in turn it will help speedup queries an order of magnitude faster over PetaBytes of data.
```
Comment (Indhumathi27) [inline docs/how-to-contribute-to-apache-carbondata.md:23]:
```suggestion
efficiency, in turn it will help speedup queries by order of magnitude faster over PetaBytes of data.
```

**Comment by Sunt-ing** [comment]
@Indhumathi27 Hello, I have updated this PR, please check it at your convenience.

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5447/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/3703/

**Comment by Indhumathi27** [comment]
@Sunt-ing Since PR-4135, is also documentation related changes, can you please keep one PR(either 4136/4135)
 for documentation change and list all the changes in the PR description

**Comment by Sunt-ing** [comment]
@Indhumathi27 Finished. And PR-4135 has been closed.

File: docs/how-to-contribute-to-apache-carbondata.md [hunk 3]
```diff
@@ -189,4 +189,4 @@ From another local branch, run:
 $ git fetch --all
 $ git branch -d <my-branch>
 $ git push <GitHub_user> --delete <my-branch>
-```
\ No newline at end of file
+```
```
Comment (Indhumathi27) [inline docs/how-to-contribute-to-apache-carbondata.md:192]: please revert this change

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/3707/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5451/

**Comment by Sunt-ing** [comment]
A little weird... should be fine now. @Indhumathi27

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/3716/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5460/

File: docs/how-to-contribute-to-apache-carbondata.md [hunk 4]
```diff
@@ -7,7 +7,7 @@
     the License.  You may obtain a copy of the License at
 
       http://www.apache.org/licenses/LICENSE-2.0
-
+
```
Comment (Indhumathi27) [inline docs/how-to-contribute-to-apache-carbondata.md:10]: Please rever this change

**Comment by Sunt-ing** [comment]
Fix done. @Indhumathi27

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5483/

**Comment by CarbonDataQA2** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/3739/

File: docs/how-to-contribute-to-apache-carbondata.md (hunk 1 above)
Comment (Indhumathi27) [reply docs/how-to-contribute-to-apache-carbondata.md:192]: Looks like it is not reverted

**Comment by Sunt-ing** [comment]
Sorry for the long-time bother, but I do think it should be fine now.  @Indhumathi27

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12602/job/ApacheCarbonPRBuilder2.3/5491/

**Comment by CarbonDataQA2** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12602/job/ApacheCarbon_PR_Builder_2.4.5/3748/

**Comment by Indhumathi27** [comment]
LGTM. Thankyou for your contribution.
